src/Display/Slider/Slider.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+
+import { Slider, Props } from './Slider';
+import { BaseContainer } from '../../Layout/GlintsContainer/GlintsContainer';
+
+export default {
+  title: 'Display/Slider',
+  component: Slider,
+  decorators: [Story => <BaseContainer>{Story()}</BaseContainer>],
+  argTypes: {
+    className: {
+      table: {
+        disable: true,
```

Comment:
Oh this is nice. I didn't know this part about controlling the display of the props!

Reply:
I think I got it from Ross ^^